blockchain/blocks/src/tipset.rs

```diff
@@ -47,6 +47,16 @@ impl PartialEq for Tipset {
     }
 }
 
+impl quickcheck::Arbitrary for TipsetKeys {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let cid = Cid::new_v1(
+            u64::arbitrary(g),
+            cid::multihash::Multihash::wrap(u64::arbitrary(g), &[u8::arbitrary(g)]).unwrap(),
+        );
+        Self { cids: vec![cid] }
```

Comment:
We also want to test what happens when you have lots of keys or zero keys.

Reply:
(You could implement this by creating a wrapper around `Cid`)

Reply:
Shouldn't that be part of a normal unit test? As far as I'm aware, these tests are just for checking if serialization/deserialization works both ways.

Reply:
I'm not sure what you mean. As the code is right now, you're just testing if serialization/de-serialization works when `TipsetKeys` contains a single `Cid`. We also want to know if it works when there are zero Cids or many Cids.